dev-site-config/navigation.config.js

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const navConfig = require('terra-dev-site/config/site/navigation.config');
+
+const providerLink = {
+  path: '/provider',
+  text: 'Provider',
+  pageTypes: ['provider'],
+  isHidden: true,
+};
```

Comment:
Changed Provider examples to use `.provider` extension to fix provider examples in terra-ui

Reply:
It's hard for us to hide primary navigation tabs when we switch terra-dev-site to use the new navigation component. I'm not sure if that's a deal breaker for this, but it is something to be aware of.